packages/atomic/src/themes/coveo.css

```diff
@@ -1,32 +1,35 @@
 :root {
-  /* Colors */
+  /* Primary colors */
   --atomic-primary: #1372ec;
-  --atomic-primary-variant: #215cd3;
+  --atomic-primary-light: #68a0ff;
+  --atomic-primary-dark: #0048b9;
```

Comment:
The light and dark aren't link to the light/dark theme concept, they are variants. If you want a different theme, you can load a different set of variables. 

It's slightly based on https://material.io/resources/color/#!/?view.left=0&view.right=0&primary.color=03A9F4&secondary.color=00897B

In our designs, the primary color is purple and the secondary color is blue, we also have the neutral colors for stuff like borders, light backgroud, alternative text.

The primary, secondary & neutral colors all have 2 variants, light & dark, those will come useful when you think of states (hover, active, etc.) In sass, you have functions like darken/lighten, but it's not compatible with us.

Here is a screenshot from the Figma
![Screen Shot 2021-05-31 at 2 33 21 PM](https://user-images.githubusercontent.com/4923043/120230371-28d7bf00-c21d-11eb-819a-b84470d94a7e.png)
Secondary for the Clear & Show More.
Primary for the checkboxes
Primary light for the hover text & checkbox.
Neutral for the border.
Neutral dark for the counts.

Reply:
I see, so more colours need to be displayed at the same time. Going from three hexs to nine for primary/secondary/neutral could be an advantage in that it provides finer control. It is more config though to avoid bad combinations, so that is something to consider.

If we want developers to be able to set any combination of the nine colours, then I would leave the PR as it is. However, if we want variants to be derived from the root colour, reducing the config back to three, perhaps something like using `filter` property could be useful:

```
.primary-light {
  background-color: var(--atomic-primary)
  filter: brightness(110%);
}

.primary-dark {
  background-color: var(--atomic-primary)
  filter: brightness(90%);
}
```

Ref: https://stackoverflow.com/questions/1625681/dynamically-change-color-to-lighter-or-darker-by-percentage-css-javascript

Reply:
I'd opt towards more control over the colors, we are not currently using atomic-primary-dark & atomic-secondary-dark, so I could remove them for now. Once that is done, we have as many colors as before, they are just renamed so they make more sense.

What we plan mid-long term is having a theme builder, similar to the material inspiration I've linked. You'd have defaults, and variants would adapt accordingly, for now we can offer our customers links to tools that quickly allow to build variant color. we would then output css variables

I don't think the filter property is equivalent as the one in sass https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/brightness(). The filter would be applied to everything (text-color, background, etc), not just a specific property, it looks like it's more for images from moz's doc.

That's how something like darken would be used, you modify the color itself before assigning it. Even here, we should have a way for customers to define specific variants, using brightness or saturation tool like this is not perfect.
```
background: darken(var(--atomic-primary), 10%)
```